compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -3098,6 +3104,17 @@ TR::Register *OMR::Power::TreeEvaluator::vloadEvaluator(TR::Node *node, TR::Code
 
    TR::LoadStoreHandler::generateLoadNodeSequence(cg, dstReg, node, opcode, 16, true);
 
```

Comment:
for this functionality, i don't think you need `vexpandbm`.  why not use `vsrab` instead, in place of `vexpandbm`?  `vsrab` is available at least from POWER7.